WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -309,8 +309,8 @@ private extension OrderListViewController {
         tableView.register(headerType.loadNib(), forHeaderFooterViewReuseIdentifier: headerType.reuseIdentifier)
     }
 
-    func configureFreeTrialBannerPresenter() {
-        self.freeTrialBannerPresenter =  FreeTrialBannerPresenter(viewController: self,
+    func configurestorePlanBannerPresenter() {
```

Comment:
Same as the other comment. 
```suggestion
    func configureStorePlanBannerPresenter() {
```

Reply:
Fixed in 88f2c7efb37ca4dff13997565f0f0c6d7c372778